lib/chef/knife/bootstrap.rb

```diff
@@ -496,6 +514,20 @@ def incomplete_policyfile_options?
         (!!config[:policy_name] ^ config[:policy_group])
       end
 
+      # True if the bootstrap version is greater than or equal to 15 or latest.
+      def bootstrap_version_gt_15?
```

Comment:
this is actually greater than or equal to, so the pattern we've used is "gte" instead of "gt" in these method names, i.e. `bootstrap_version_gte_15?`